advice-dont-ask-about-solutions.md

```diff
@@ -0,0 +1,55 @@
+# Don't ask about solutions[^1]
+
+You may have been sent this document because **you asked about solutions** to a problem,
+rather than the problem itself, or without explaining your original problem.
```

Comment:
```suggestion
You may have been sent this document because **you asked about solutions** to a problem, rather than the problem itself, or without explaining your original problem.
```

imo better to let the viewer wrap text